schemas/IDLs/org.gel.models.report.avro/6.0.0/ExitQuestionnaire.avdl

```diff
@@ -0,0 +1,360 @@
+        Only applicable if it is necessary to design a new assay to validate the variant.
+        */
+        union {null, double} validation_assay_time;
+
+        /**
+        Technical Laboratory Validation. Total time for validation test wet work for all variants (hours).
+        */
+        union {null, double} wet_validation_time;
+
+        /**
+        Analytical Laboratory Validation. Total time for analysis of validation results for all variants (hours).
+        */
+        union {null, double} analytical_validation_time;
+
+        /**
+        Primary Reporting. Time taken to complete primary reporting stage (hours).
+        */
+        double primary_reporting_time;
+
+        /**
+        Report Authorisation. Time taken to check and authorise report (hours).
+        */
+        double primary_authorisation_time;
+
+        /**
+        Report Distribution.
+        Please enter, where possible/accessible how long it takes for the result to be conveyed to the patient.
+        E.g. via letter from the clinician (days).
+        */
+        double report_distribution_time;
+
+        /**
+        Total time from result to report.
+        The total time taken from when the analysis of the WGS results started  to a report being received
+        by the patient include any 'waiting' time (days).
+        */
+        double total_time;
+
+        /**
+        Which parts of the WGA were reviewed?
+        */
+        ReviewedParts reviewedInMdtWga;
+
+        /**
+        Were potentially actionable variants detected?
+        */
+        CancerActionableVariants actionableVariants;
+    }
+
+    /**
+    The questions for the cancer program exit questionnaire for germline variants
+    */
+    record CancerGermlineVariantLevelQuestions{
+        /**
+        Variant coordinates following format `Chr: Pos Ref > Alt`
+        */
+        // TODO: use VariantCoordinates in CommonInterpreted.avdl
+        string variantDetails;
+
+        /**
+        How has/will this potentially actionable variant been/be used?
+        */
+        CancerUsabilityGermline variantUsability;
+
+        /**
+        Has this variant been tested by another method (either prior to or following receipt of this WGA)?
+        */
+        CancerTested variantTested;
+
+        /**
+        Please enter validation assay type e.g Pyrosequencing, NGS panel, COBAS, Sanger sequencing. If not applicable enter NA;
+        */
+        string validationAssayType;
+    }
+
+    /**
+    The questions for the cancer program exit questionnaire for somatic variants
+    */
+    record CancerSomaticVariantLevelQuestions{
+        /**
+        Variant coordinates following format `Chr: Pos Ref > Alt`
+        */
+        // TODO: use VariantCoordinates in CommonInterpreted.avdl
+        string variantDetails;
+
+        /**
+        Type of potential actionability:
+        */
+        CancerActionability variantActionability;
```

Comment:
@Antonior26 the model in 5 is:
```
array<CancerActionabilitySomatic> variantActionability;
```
so have the arrays always just been of one `CancerActionabilitySomatic` object ?